test/fixtures/gateway-account.fixtures.js

```diff
@@ -73,6 +97,11 @@ function validGatewayAccount (opts) {
     gatewayAccount.worldpay_3ds_flex = validWorldpay3dsFlexCredentials(opts.worldpay_3ds_flex)
   }
 
+  // provider switch enabled is only available to the "frontend" resource, it isn't always guaranteed
+  if (opts.provider_switch_enabled) {
```

Comment:
We might want to set to false in the opts? should this check if it is undefined?

Reply:
I originally had this set to opts or default `false` but it fails pacts because the confusing `/v1/api` routes won't return the value, I guess as it fails the same fixture is used for both